pyface/tests/test_split_panel.py

```diff
@@ -47,8 +47,7 @@ def test_lifecycle(self):
         self.assertIsNone(self.widget.control)
 
         with self.event_loop():
-            self.widget.parent = self.window.control
-            self.widget.create()
+            self.widget.create(self.window.control)
```

Comment:
Similarly here I think it would help readability to have an explicit `parent=self.window.control`

Reply:
I can see the point here.  I'll make a change.